mu4/notation/qml/MuseScore/NotationScene/internal/NotationToolBarButton.qml

```diff
@@ -0,0 +1,89 @@
+import QtQuick 2.8
```

Comment:
Can we just improve FlatButton to add an ability add icon to right side of a text?